templates/webpack.config.js

```diff
@@ -0,0 +1,53 @@
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
+const HtmlPlugin = require('html-webpack-plugin');
+const path = require('path');
+
+const isProduction = process.env.NODE_ENV === 'production';
+
+module.exports = {
+  entry: './src/scripts/index.js',
+  output: {
+    filename: 'bundle.js',
+    path: path.resolve(__dirname, 'build')
+  },
+  devServer: {
+    contentBase: './build',
+    port: 8000
+  },
+  devtool: isProduction ? false : 'source-map',
+  module: {
+    loaders: [
+      {
+        test: /\.jsx?$/,
+        exclude: /node_modules/,
+        loader: 'babel-loader',
+        query: {
+          presets: ['env'],
+          plugins: ['transform-object-rest-spread']
```

Comment:
I'd argue that we use this over `Object.assign` in most of the current projects, especially with `redux` in the stack.